lib/errors/sql-errors.js

```diff
@@ -1,10 +1,10 @@
-var util = require('util');
+var util = require("util");
 
 /**
  * Specific error for this library so we can tell it apart from other errors.
  * @class
- * @param {String} message
- * @param {String} [key]
+ * @param {String} message   error message
```

Comment:
Yeah, could be more descriptive and value adding.  But each jsdoc param should have a description to be valid and it seemed unoffensive.